src/mmw/js/src/core/templates/weatherStationPopup.html

```diff
@@ -0,0 +1,19 @@
+<div class="data-catalog-popover-result-region">
+    <div class="resource-wrapper">
+        <h3 class="resource-title">
+            Weather Station: {{ station_id }}</h2>
+        </h3>
+        <div class="resource-detail">
+            <i class="fa fa-star"></i> {{ location if location else '' }}
+        </div>
+        <div class="resource-detail">
+            <i class="fa fa-calendar"></i> {{ beginYear }} - {{ endYear|toDateWithoutTime }}
```

Comment:
Similarly, let's have a symmetrical treatment of dates here. Let's do just the year in both cases.